.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,31 +1,27 @@
 ##### SUMMARY
-<!--- Describe the change, including rationale and design decisions -->
+<!--- Describe the change below, including rationale and design decisions -->
 
-<!--- If you are fixing an existing issue, please include "Fixes #nnn" in your
-commit message and your description; but you should still explain what
-the change does.-->
+<!--- HINT: Include "Fixes #nnn" if you are fixing an existing issue -->
 
 ##### ISSUE TYPE
-<!--- Pick one below and delete the rest: -->
- - Feature Pull Request
```

Comment:
Not sure if you can change these, the bot may complain

Reply:
Fixed.